test/integration/processing/calculation/get-calculation.test.js

```diff
@@ -44,14 +44,17 @@ describe('process get calculation object', () => {
     expect(wrapper).rejects.toThrow()

```

Comment:
Couple things picked up:

1. Test `describe` says payment request object not calculation object
2. in `beforeEach` we have the mock-object pointing to the `retreivedCalculationData` i.e. the mock calc obj refers to the extracted calculation record we extract, this is not the purpose of these mock objects. A mock calculation object should model the mock calculation message coming from DWH and therefore, sim. to line 17 in this file the `retreivedCalculation` should then use this to subset what we acc'ly want to extract from the db and that is what we validate against -- currently our `schema.validate` does not mimic what the actual schema is checking for, i.e. our mocked accepts an obj w/ `sbi`, `calculated`, `invoiceNumber` and `paymentRequestId` we are missing `calculationId`

- also refers to another comment on mine but we have 2 mock calculation objects -- can we not merge or use the 1?

Reply:
Also missing tests re: `updateCalculationPaymentRequestId`